automation/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
should this config file be log4j or logback?

Reply:
For what it's worth, the last time I checked, logback was the long-term maintained solution.